inductiva/resources/machines.py

```diff
@@ -124,4 +132,6 @@ def _log_machine_group_info(self):
         logging.info("Number of machines: %s", self.num_machines)
         logging.info("Spot: %s", self.spot)
         logging.info("Disk size: %s GB", self.disk_size_gb)
-        logging.info("Estimated cost per hour: %s $/h", self.estimated_price)
+        # We won't offer price values just yet, due to a bug.
+        # TODO: Fix the price computation.
+        # logging.info("Estimated cost per hour: %s $/h", self.estimated_price)
```

Comment:
Why is this change introduced here? Do it in a separate PR, to describe the bug, etc?

Reply:
This change is done here, since it is related with the new MachineGroup class being deployed and does not let machines to be launched.